From d0529275cf70aa60b7942f0306217913df6bf53e Mon Sep 17 00:00:00 2001
From: Matthias Coppens <coppens.matthias.abc@gmail.com>
Date: Wed, 26 Aug 2020 14:24:36 +0200
Subject: [PATCH 3/3] SConstruct Python 3 fix

`dict.has_key` is removed in Python 3.X, use `in` instead.
---
 SConstruct | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git SConstruct SConstruct
index f20e1745..0e7f02e9 100644
--- SConstruct
+++ SConstruct
@@ -140,11 +140,11 @@ else:
         CC = 'gcc', CXX = 'g++',
         options = opts)
     # Take environment variables into account
-    if os.environ.has_key('CXX'):
+    if 'CXX' in os.environ:
         env['CXX'] = os.environ['CXX']
-    if os.environ.has_key('CXXFLAGS'):
+    if 'CXXFLAGS' in os.environ:
         env['CXXFLAGS'] += SCons.Util.CLVar(os.environ['CXXFLAGS'])
-    if os.environ.has_key('LDFLAGS'):
+    if 'LDFLAGS' in os.environ:
         env['LINKFLAGS'] += SCons.Util.CLVar(os.environ['LDFLAGS'])
     check_headers = ['GL/gl.h', 'SDL2/SDL.h', 'SDL2/SDL_image.h', 'vorbis/vorbisfile.h', 'curl/curl.h', 'bullet/btBulletCollisionCommon.h', 'bullet/btBulletDynamicsCommon.h']
     check_libs = []
-- 
2.29.0.rc1

